Well, there goes my entire weekend... ^^ (This is the correct one)

RandomPianist opened the pull request:
THIS IS AN UPDATED VERSION OF MY PULL REQUEST

I saw this repo at saturday morning and I felt the urge of dive myself in async JavaScript.
Now it's Sunday 11pm (now Monday 6pm) in Brazil and I do not regret anything. Maybe I should eat something lol

Anyway, your application is superb, but I dare propose a few ameliorations.

## Improvements

- Deleted unuseful commented code. All comments in this version are documentation-relative.
- Improved code presentation: spacing and identation remade.
- Fixed a bug regarding the long exposure of the leader card.
- Fixed typo in page title.
- Fixed the style of end screen buttons.
- Fixed notification read times.
- Added sound effects to the game.

#### Changes
- The game is now played in full screen.
- As in the original game, passing the round now requires a countdown timer.
- As in the original game, modifying cards in the deck now requires double-clicking (or pressing **"Enter"** key [still in alfa]).

### Operations included with the keyboard:
**"E"** starts the game
**"X"** uses or modifies the leader card
**"Q"** closes the card explanation windows
**"Space"** passes the current round
**"Enter"** plays the cards
**Arrows** select cards on the carousel

RandomPianist committed:
Update README.md

RandomPianist committed:
Add files via upload

RandomPianist committed:
Update README.md

RandomPianist committed:
Update README.md

RandomPianist committed:
Update README.md

RandomPianist committed:
Add files via upload

RandomPianist committed:
Add files via upload

RandomPianist committed:
Add files via upload

RandomPianist committed:
Add files via upload

RandomPianist committed:
Add files via upload

RandomPianist committed:
Line break at some notifications

RandomPianist committed:
Add a load text

RandomPianist committed:
Add files via upload

RandomPianist committed:
Hides screens while they're not loaded

RandomPianist committed:
Add files via upload

RandomPianist committed:
Add notifications about leader movements

RandomPianist committed:
Add files via upload

RandomPianist committed:
"Nilfgaard wins draws" notification added

RandomPianist committed:
Add notif_nilfgaard.png

RandomPianist committed:
Add images to a few notifications

Add images to remanescent faction-related notifications

RandomPianist committed:
Better "pass-round" button system

RandomPianist committed:
Right click on pass button stops counting

RandomPianist committed:
Correction on leader calling

RandomPianist committed:
New correction on leader calling

RandomPianist committed:
File extension correction

File extension correction on faction-relative notifications.

RandomPianist committed:
Notification correction

RandomPianist committed:
Enable pass-button when playing on a mobile device

RandomPianist committed:
Click cards play them

When possible, clicking on some card which is being displayed in details will play it.

RandomPianist committed:
Correct bug in leader selection

RandomPianist committed:
Correct bug in leader selection

einstein95 commented on gwent.js:
```diff
@@ -1687,35 +1928,90 @@ class UI {
 		this.previewCard = null;
 		this.lastRow = null;
-		document.getElementById("pass-button").addEventListener("click", () => player_me.passRound(), false);
+		if (!navigator.userAgentData.mobile) {
```
This breaks compatibility with Firefox. Something like `if (!("ontouchstart" in document.documentElement))` solves this problem

RandomPianist committed:
Update gwent.js

Resumed compatibility with Firefox, under suggestion from @einstein95

RandomPianist committed:
Delete img directory

Deleting in order to upload new file tree

RandomPianist committed:
Delete sfx directory

Deleting in order to upload a new file tree

RandomPianist committed:
Delete LICENSE

Deleting in order to upload a new file tree

RandomPianist committed:
Delete style.css

Deleting in order to upload a new file tree

RandomPianist committed:
Delete README.md

Deleting in order to upload a new file tree

RandomPianist committed:
Delete abilities.js

Deleting in order to upload a new file tree

RandomPianist committed:
Delete cards.js

Deleting in order to upload a new file tree

RandomPianist committed:
Delete decks.js

Deleting in order to upload a new file tree

RandomPianist committed:
Delete factions.js

Deleting in order to upload a new file tree

RandomPianist committed:
Delete gwent.js

Deleting in order to upload a new file tree

RandomPianist committed:
Files from new version

Adds: 1 - a page for rules (by @romain-durban, author of version 3.0), 2 - an A.I. vs A.I. option (by @romain-durban, author of version 3.0), 3 - a statistics table, 4 - a "give up" button

RandomPianist committed:
Files from new version

Adds: 1 - a page for rules (by @romain-durban, author of version 3.0), 2 - an A.I. vs A.I. option (by @romain-durban, author of version 3.0), 3 - a whole new set of cards and mechanics (by @romain-durban, author of version 3.0), 4 - a statistics table, 5 - a "give up" button, 6 - new visuals for carousel.
Corrects: 1 - aesthetic values, 2 - unused code, 3 - bug in leader selection, 4 - bug when drawing a match, 5 - typo errors, 6 - bug when redrawing cards at the start of each match.

RandomPianist committed:
Resources for new version (includes new favicon)

RandomPianist committed:
Resources for new version (includes new decks)

RandomPianist committed:
Resources for new version (includes new abilities)

RandomPianist committed:
Resources for new version (includes rules)

RandomPianist committed:
Resources for new version (includes new cards)

RandomPianist committed:
Resources for new version (includes new cards)

RandomPianist committed:
Resources for new version (includes new cards)

RandomPianist committed:
Resources for new version (includes new cards)

RandomPianist committed:
Resources for new version (includes new cards)

RandomPianist committed:
Resources for new version (includes new cards)

RandomPianist committed:
Resources for new version (includes new cards)

RandomPianist committed:
Resources for new version (includes new cards)

RandomPianist committed:
Resources for new version (includes new cards)

RandomPianist committed:
Resources for new version (includes new cards)

RandomPianist committed:
Resources for new version (includes new cards)

RandomPianist committed:
Resources for new version (includes new cards)

RandomPianist committed:
Resources for new version (includes new cards)

RandomPianist committed:
Informs the changes in this version

RandomPianist committed:
Informs changes in this version

RandomPianist committed:
Resources for new version

RandomPianist committed:
Reorder changes

RandomPianist committed:
Removes unused code

RandomPianist committed:
Reorder cards

Cards in deck selector are now shown in alphabetical order
